docs/DeveloperGuide.md

```diff
@@ -1470,6 +1504,21 @@ testers are expected to do more *exploratory* testing.
       size of student list)<br>
       Expected: No student is deleted. Error details shown in the status message.
 
+#### Finding a student
+
+1. Finding a student.
+
+   1. Prerequisites: Student names that contain `Alex` and `Yu`. The sample data of Tutor's Pet contains such names and can be used.
+
+   1. Test case: `find-student alex` \
+      Expected: All students whose name contains `alex` (case-insensitive) should be displayed.
+
+   1. Test case: `find-student alex yu` \
+      Expected: All students whose name contains `alex` or `yu` (case-insensitive) should be displayed.
+
+   1. Incorrect find command to try: `find-student`. \
+      Expected: The displayed student list does not get updated. Error details shown in the status message.
```

Comment:
Perhaps it would be better to keep the format same as the previous few?
```suggestion  
    1. Other incorrect commands to try:
       * `find-student`
       Expected: The displayed student list does not get updated. Error details shown in the status message.
```